randovania/generator/old_generator_reach.py

```diff
@@ -60,7 +61,7 @@ class OldGeneratorReach(GeneratorReach):
     _reachable_costs: dict[int, int] | None
     _node_reachable_cache: dict[int, bool]
     _unreachable_paths: dict[tuple[int, int], RequirementSet]
-    _safe_nodes: set[int] | None
+    _safe_nodes: tuple[list[int], set[int]] | None
```

Comment:
would it hurt performance to use a NamedTuple here for readability's sake? if it would cause too much trouble then it's ok as-is tho

Reply:
tbh I'm not even sure of how relevant this change is